extendr-macros/src/output_r.rs

```diff
@@ -118,6 +118,15 @@ fn write_r_wrapper(wrapper_fn: WrapperFn) -> Result<(), Box<dyn Error>> {
 
     let file = File::create(&output_file_path)?;
     let mut writer = BufWriter::new(file);
+    write!(
+        &mut writer,
+        r#"
+===========
+THIS FILE IS AUTOGENERATED VIA #[extendr] MACROS
+DO NOT ATTEMPT TO EDIT IT BY HAND
+===========
```

Comment:
Could you add a `#` in front of these four lines? The generated file is meant to be read as R code and so this warning notice needs to be formatted as an R comment.